tests/phpunit/unit/api/utilities/beansUrlToPath.php

```diff
@@ -21,13 +21,23 @@
  */
 class Tests_BeansUrlToPath extends Test_Case {
 
+	/**
+	 * Sanitized ABSPATH - to ensure it works on both OSX and Windows.
+	 *
+	 * @var string
+	 */
+	protected static $abspath;
+
 	/**
 	 * Setup test fixture.
 	 */
 	protected function setUp() {
 		parent::setUp();
 
 		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
+		if ( is_null( self::$abspath ) ) {
```

Comment:
@hellofromtonya now that the `ABSPATH` is sanitized globally here, this shouldn't be needed and all `self::$abspath` should be replaced back to `ABSPATH`.

PS: I haven't tested it but it should be the case.

Reply:
@ThierryA Actually no.  We still need that. Why?  The `ABSPATH` is the real fully qualified path on the computer's filesystem.  In our API, we need to get file access for our conversions as well as loading files, such as in the Compiler.  

In the conversions, as shown above, we need to strip off the Windows drive, e.g. C:, D:, E:, etc.  

Looking through the API, we need to retain the true `ABSPATH`, as it is in WordPress, meaning pointing to the root path of the website filesystem.  Why? We are loading a file from WordPress in the Compiler.  Plus, in the future, we may need to use `ABSPATH` for other purposes for filesystem access.

Therefore, I'm going to change `ABSPATH` to be exactly what it is in WordPress, instead of referencing the tests directory as I originally set it up.  That will save us a bunch of time and frustrations down the road.

I'm adjusting the tests now and will push the new commit.